processing/src/main/java/org/apache/carbondata/processing/sort/sortdata/SortParameters.java

```diff
@@ -37,6 +40,13 @@
 import org.apache.log4j.Logger;
 
 public class SortParameters implements Serializable {
+  
+  private ExecutorService writeService = Executors.newFixedThreadPool(5,
```

Comment:
Suggest to make it configurable when set core pool size for threadpool

Reply:
ok,done.